examples/plugins/traffic-control/report.go

```diff
@@ -224,7 +224,7 @@ func getControls() []extControl {
 				Rank:  22,
 			},
 			handler: func(pid int) error {
-				return DoTrafficControl(pid, 1)
+				return DoTrafficControl(pid, "1ms")
```

Comment:
This is already the format that `tc` wants. Because this are hard coded values, I think it is ok for now.